db/seeds.rb

```diff
@@ -10,43 +10,15 @@
 require 'ffaker'
 expirations = ((Date.today - 7.days)..(Date.today + 6.months)).to_a
 
-10.times do
-  Passenger.create! name: FFaker::Name.name,
-                    address: FFaker::Address.street_address,
-                    email: FFaker::Internet.email,
-                    phone: FFaker::PhoneNumber.short_phone_number,
-                    wheelchair: false,
-                    active: true,
-                    permanent: false,
-                    expiration: expirations.sample
-  Passenger.create! name: FFaker::Name.name,
-                    address: FFaker::Address.street_address,
-                    email: FFaker::Internet.email,
-                    phone: FFaker::PhoneNumber.short_phone_number,
-                    wheelchair: false,
-                    active: true,
-                    permanent: true
+20.times do
```

Comment:
Okay. I ask because covering all your bases is good for seed data, but so is resembling production data. It seemed like a weird distribution of users that wouldn't actually exist. In particular, having the vast majority of your users having extremely proximate expiration dates doesn't seem quite right.